before-notebook.d/41_setup-hq-computer.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -x
+
+# XXX: need to make daemon start late in docker stack
+# https://github.com/aiidalab/aiidalab-docker-stack/pull/498
+verdi daemon stop || echo "stop fail"
+
+computer_list=$(verdi computer list)
+if echo ${computer_list} | grep -q ${HQ_COMPUTER}; then
+    echo "${HQ_COMPUTER} already setup"
+else
+    # computer
+    verdi computer show ${HQ_COMPUTER} || verdi computer setup        \
```

Comment:
I don't think you need this branch at all. You can just use the `verdi computer show` command to determine if the computer exists or not.

```suggestion
verdi computer show ${HQ_COMPUTER} || verdi computer setup        \
```

Reply:
worth to try, I'll change it.